zipline/assets/futures.py

```diff
@@ -180,3 +180,84 @@ def offset(self, time_delta):
 
         """
         return self.as_of(self.as_of_date + Timedelta(time_delta))
+
+
+# http://www.cmegroup.com/product-codes-listing/month-codes.html
+CME_CODE_TO_MONTH = dict(zip('FGHJKMNQUVXZ', range(1, 13)))
+MONTH_TO_CME_CODE = dict(zip(range(1, 13), 'FGHJKMNQUVXZ'))
+
+
+def cme_code_to_month(code):
+    """
+    Convert a CME month code to a month index.
+
+    The month codes are as follows:
+
+    'F' -> 1  (January)
+    'G' -> 2  (February)
+    'H' -> 3  (March)
+    'J' -> 4  (April)
+    'K' -> 5  (May)
+    'M' -> 6  (June)
+    'N' -> 7  (July)
+    'Q' -> 8  (August)
+    'U' -> 9  (September)
+    'V' -> 10 (October)
+    'X' -> 11 (November)
+    'Z' -> 12 (December)
+
+    Parameters
+    ----------
+    code : str
+        The month code to look up.
+
+    Returns
+    -------
+    month : int
+       The month number (starting at 1 for January) corresponding to the
+       requested code.
+
+    See Also
+    --------
+    month_to_cme_code
+        Inverse of this function.
+    """
+    return CME_CODE_TO_MONTH[code]
+
+
+def month_to_cme_code(month):
+    """
+    Convert a month to a CME code.
+
+    The month codes are as follows:
+
+    'F' -> 1  (January)
```

Comment:
Suggest changing the doc string here to read `1 -> 'F' (January)`,  etc., so that it illustrates what the method does.